audits/README.md

```diff
@@ -0,0 +1,14 @@
+# <img src="../logo.svg" alt="Balancer" height="128px">
+
+# Balancer V3 Audits
+
+This directory the reports of audits performed on Balancer smart contracts by different security firms.
+The audits have been conducted before the [Cantina contest](https://cantina.xyz/competitions/949ad7c5-ea14-427d-b10a-54e33cef921b), and all the relevant findings addressed.
+
+| :warning: | Audits are not a guarantee of correctness. Some of the contracts were modified after they were audited.      |
+| --------- | :----------------------------------------------------------------------------------------------------------- |
+
+| Scope                             | Firm          | Report                                         |
+| --------------------------------- | ------------- | ---------------------------------------------- |
+| Vault, Weighted Pool, Stable Pool | Certora       | [`2024-09`](./certora/2024-09.pdf)             |
+| Vault, Weighted Pool, Stable Pool | Trail Of Bits | [`2024-10`](./trail-of-bits/2024-10.pdf)       |
```

Comment:
I don't have a strong preference, but we did use the full date in the corresponding v2 file. Would be 2024-09-04 for Certora, 2024-10-08 for ToB.

Reply:
We're waiting on Spearbit as noted. What about the 4th one?

Reply:
Could be, yes. Merging this one for now, but it could be interesting to push that one too.